src/centerpoint.jl

```diff
@@ -0,0 +1,16 @@
+"""
+    CenterPoint(dims) -> cp
+
+Create a fixed point which can be used in `imresize` and `imrotate`
+functions in order to keep the value in this point the same, i.e.,
+
+```jldoctest
+img[cp] == imgr[cp]
+```
```

Comment:
Oh, I could make such a bad mistake, sorry. I will test it carefully.

Reply:
Didn't notice '_at all_' and got the meaning of your comment wrong.
> All we're introducing is a warp center point.

Honestly, that's what I was thinking about. Not to disturb your state, but can you give any tip of how this idea looks in your head? Othewise I'm afraid I don't understand.

Reply:
Sorry for the back-and-forth and the inaccuracy.

In https://evizero.github.io/Augmentor.jl/dev/operations/#Affine-Transformations, the scale/resize is no different from `imresize`. I was thinking of a generic solution to the `Zoom` operator there where the center point is invariant. The term "fixed point" showed itself in my head when I wrote down #121.

Reply:
Sorry for the delayed response, got stuck with exams. Unfortunately, I still can't fully understand your concerns. I made a poor visualization of what it does right now when I create a new image with `imresize(img, FixedPoint(100, 100); ratio=0.5)`.
![resizing](https://user-images.githubusercontent.com/42866169/123509195-b2dc4180-d67c-11eb-9891-ba85aea36c02.png)
To me it looks like it works as expected and implements the `Zoom` operator, except it doesn't crop the image. Hopefully, my point is clear and makes sense.

Reply:
Looks good to me! I'll recheck next weekend (also busy with school stuff). In the meantime, if you can add up some tests when you're available it would be great.